src/base/utils/CoreModel.jl

```diff
@@ -17,3 +17,46 @@ Base.isequal(model1::CoreModelCoupled, model2::CoreModelCoupled) =
     isequal(model1.cu, model2.cu)
 
 Base.copy(model::CoreModelCoupled) = CoreModelCoupled(model.lm, model.C, model.cl, model.cu)
+
+"""
+Returns indices of exchange reactions.
+Exchange reactions are identified based on most commonly used prefixes.
+"""
+function find_exchange_reactions(
+    model::CoreModel;
+    exclude_biomass = false,
+    biomass_str::String = "biomass",
+    exc_prefs = ["EX_"; "Exch_"; "Ex_"; "R_EX_"],
```

Comment:
Should we include all versions with the added "R_"? Also, maybe this list could just be put in `constants.jl`?

Reply:
ah, yeah we probably should. I will do it now. @exaexa is going to hate this elaborate guessing game :)

Reply:
As long as he's hating the game and not the players...